website/src/content/concepts/merchant-center-api.mdx

```diff
@@ -0,0 +1,172 @@
+
+Therefore, Custom Applications use the Merchant Center API to make requests to one of the commercetools Platform APIs. The way you select the API in your requests depends on the [API Gateway Endpoints](#api-gateway-endpoints).
+
+![Merchant Center API](/images/merchant-center-api.png 'Merchant Center API')
+
+# Cloud regions
+
+The commercetools Platform is available in [multiple cloud regions](https://docs.commercetools.com/api/general-concepts#regions).
+These regions are completely isolated from each other and no data is transferred between them.
+
+<Info>
+
+Platform accounts created for one region are not valid for the other regions. A [signup](https://docs.commercetools.com/tutorials/getting-started) is required for each region individually.
+In case you need advice in which region your Project should be located, please contact [commercetools support](https://support.commercetools.com).
+
+</Info>
+
+## Hostnames
+
+The Merchant Center and the Merchant Center API Gateway are available at the same cloud regions where the commercetools Platform runs.
+
+All hostnames are subdomains of `commercetools.com` and follow a specific naming format, including the cloud provider, the cloud region, and the Merchant Center service name.
+
+```
+https://{mcService}.{region}.{cloudProvider}.commercetools.com
+```
+
+- `mcService`: for the Merchant Center front-end the value is `mc`, for the Merchant Center API Gateway the value is `mc-api`.
+- `region`: the region of the cloud provider, see table below.
+- `cloudProvider`: the cloud provider, either `gcp` or `aws`.
+
+| Cloud region | Merchant Center API Gateway hostname |
+| --- | --- |
+| Australia (Google Cloud, Sydney) | `mc-api.australia-southeast1.gcp.commercetools.com` |
+| Europe (Google Cloud, Belgium) | `mc-api.europe-west1.gcp.commercetools.com` |
+| Europe (AWS, Frankfurt) | `mc-api.eu-central-1.aws.commercetools.com` |
+| North America (Google Cloud, Iowa) | `mc-api.us-central1.gcp.commercetools.com` |
+| North America (AWS, Ohio) | `mc-api.us-east-2.aws.commercetools.com` |
+
+## Cloud Identifiers
+
+To make it easier to reference the Merchant Center API URL, for example in the [Custom Application config](/api-reference/application-config#cloudIdentifier), each cloud region maps to an identifier.
+
+| Cloud region | Cloud identifier |
+| --- | --- |
+| Australia (Google Cloud, Sydney) | `gcp-au` |
+| Europe (Google Cloud, Belgium) | `gcp-eu` |
+| Europe (AWS, Frankfurt) | `aws-fra` |
+| North America (Google Cloud, Iowa) | `gcp-us` |
+| North America (AWS, Ohio) | `aws-ohio` |
+
+# Authentication
+
+The Merchant Center API is protected by a **session token** via the HTTP `Cookie` header, which is set only for `<cloud-region>.commercetools.com` domains.
+
+In the browser, the session token is stored in a **secure cookie** named `mcAccessToken`.
```

Comment:
Maybe here we might say that this happens in the local environment?
I mean, the HTTP cookie also gets stored in the browser.

Reply:
Hmm what do you mean? I already added a note below mentioning the difference in local development.

Reply:
Oh, sorry; I missed the comment below 😞